apis/apps/v1alpha1/opsrequest_webhook.go

```diff
@@ -416,14 +416,74 @@ func (r *OpsRequest) validateHorizontalScaling(_ context.Context, _ client.Clien
 	if len(horizontalScalingList) == 0 {
 		return notEmptyError("spec.horizontalScaling")
 	}
-
+	for _, v := range r.Spec.HorizontalScalingList {
+		if v.Operator != HScaleDeleteOP {
```

Comment:
why not validate if v.Operator equals to HScaleAddOP or HScaleOverwriteOP?

for example, if `autoSyncReplicas` is true, `replicas` should be empty.

Reply:
fixed